grodecoder.py

```diff
@@ -1317,7 +1317,7 @@ def is_met(graph: nx.classes.graph.Graph) -> bool:
 def is_nucleic_acids(graph: nx.classes.graph.Graph) -> bool:
     set_key_amino_acid_mda = set(mol_def.NUCLEIC_ACIDS.keys())
```

Comment:
The variable name `set_key_amino_acids_mda` does not seem appropriate. Could you use `nucleic_acid_resnames` instead?